src/native/libs/System.Security.Cryptography.Native/pal_evp_kem.h

```diff
@@ -6,16 +6,19 @@
 #include "opensslshim.h"
 
 PALEXPORT int32_t CryptoNative_EvpKemAvailable(const char* algorithm);
+PALEXPORT int32_t CryptoNative_EvpKemGetName(const EVP_PKEY* pKey, char* algorithm, int32_t* algorithmLength);
 PALEXPORT EVP_PKEY* CryptoNative_EvpKemGeneratePkey(const char* kemName, uint8_t* seed, int32_t seedLength);
 PALEXPORT int32_t CryptoNative_EvpKemExportPrivateSeed(const EVP_PKEY* pKey, uint8_t* destination, int32_t destinationLength);
 PALEXPORT int32_t CryptoNative_EvpKemExportDecapsulationKey(const EVP_PKEY* pKey, uint8_t* destination, int32_t destinationLength);
 PALEXPORT int32_t CryptoNative_EvpKemExportEncapsulationKey(const EVP_PKEY* pKey, uint8_t* destination, int32_t destinationLength);
 PALEXPORT int32_t CryptoNative_EvpKemEncapsulate(EVP_PKEY* pKey,
+                                                 void* extraHandle,
                                                  uint8_t* ciphertext,
                                                  int32_t ciphertextLength,
                                                  uint8_t* sharedSecret,
                                                  int32_t sharedSecretLength);
 PALEXPORT int32_t CryptoNative_EvpKemDecapsulate(EVP_PKEY* pKey,
+                                                void* extraHandle,
```

Comment:
This looks one space shy of aligned?

```suggestion
                                                 void* extraHandle,
```

Reply:
IDEs and clang format get along super well.